test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/Common/Entity/HttpRequestEntity.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.Common
+{
+    // Custom representation of Http Requests.
+    public class HttpRequestEntity
+    {
+        public string Url { get; private set; }
+        public string HttpMethod { get; private set; }
+        private readonly Dictionary<string, string> _headers;
+        private readonly Dictionary<string, string> _requestParams;
+        private readonly string _requestBody;
+
+        public HttpRequestEntity(string url, string httpMethod, Dictionary<string, string> headers, 
+            Dictionary<string, string> requestParams, string requestBody)
+        {
+            Url = url;
+            HttpMethod = httpMethod;
+            _headers = headers;
+            _requestParams = requestParams;
+            _requestBody = requestBody;
+        }
+
+        public string GetUrlWithQuery()
+        {
+            StringBuilder stringBuilder = new StringBuilder(Url);
+            stringBuilder.Append("?");
+            stringBuilder.Append(GetQuery());
+            
+            return stringBuilder.ToString(); 
+        }
+        private string GetQuery()
+        {
```

Comment:
Single line between functions. You can use Visual Studio feature to format the whole file.

Reply:
Unfortunately, the IDE did not take care of this automatically. Checked for proper spacing throughout the code.